pkg/lib/userexport/model.go

```diff
@@ -226,3 +231,77 @@ func NewResponseFromTask(task *redisqueue.Task) (*Response, error) {
 
 	return response, nil
 }
+
+func jsonPointerToCSVFieldName(pointer string) (fieldName string) {
+	splitFn := func(c rune) bool {
+		return c == '/'
+	}
+	fields := strings.FieldsFunc(pointer, splitFn)
+	return strings.Join(fields, ".")
```

Comment:
You can just parse the pointer with `jsonpointer.Parse`, it gives you a slice of strings. Then you call `strings.Join(referenceTokens, ".")`. Write test for this function.

Reply:
updated